src/core/src/op/concat.cpp

```diff
@@ -52,42 +52,56 @@ std::shared_ptr<Node> Concat::clone_with_new_inputs(const OutputVector& new_args
     return std::make_shared<Concat>(new_args, m_axis);
 }
 
+inline std::vector<const void*> get_in_buffers(const TensorVector& inputs) {
```

Comment:
This function can be used on just tensors, their purpose is outer scope perspective.
```suggestion
inline std::vector<const void*> get_buffers(const TensorVector& tensors) {
```

Reply:
This function can be removed and create this vector can be done in op's evalute where inputs shapes are collected.

Reply:
Removed